### PR TITLE
Media icon bl

### PR DESCRIPTION
**Pull request by blongden73**
## What does this change?
Fixes a problem with quotes in the headline of articles, on media cards. 

Removes some CSS in the process. :-)

### CODE REVIEW

**Review by zeftilldeath — APPROVED** [review]
🤙

File: static/src/stylesheets/module/facia-garnett/item-types/_fc-item--type-media.scss [hunk 1]
```diff
@@ -1,41 +1,16 @@
...
     background-color: $media-garnett-main-1;
 
-    .inline-icon__svg:not(.inline-clock__svg) {
-        padding: 5px;
-        border-radius: 50%;
-    }
+    // .inline-icon__svg:not(.inline-clock__svg) {
+    //     padding: 5px;
+    //     border-radius: 50%;
+    // }
```
Comment (SiAdcock) [inline static/src/stylesheets/module/facia-garnett/item-types/_fc-item--type-media.scss]: Is it safe to delete this?

**Review by SiAdcock — APPROVED** [review]
One tiny thing, otherwise 👍

File: static/src/stylesheets/module/facia-garnett/item-types/_fc-item--type-media.scss (hunk 1 above)
Comment (blongden73) [reply static/src/stylesheets/module/facia-garnett/item-types/_fc-item--type-media.scss]: ahh yeah, this was the initial problem, i'll delete it!

**Pull request merged**